### PR TITLE
Fix extra flags

### PR DESCRIPTION
**Pull request by h314to**

### CODE REVIEW

**Comment by h314to** [comment]
Try using eval outside of the makefile. Still convoluted, but at least you should be able to expand the variables.

File: .circleci/config.yml [hunk 1]
```diff
@@ -165,15 +165,15 @@ references:
       - <<: *update_versions
       - deploy:
           name: Install Codacy
-          command: make -e -C .doks/ deploy_to_doks_from_local_dir
+          command: eval make -e -C .doks/ deploy_to_doks_from_local_dir HELM_INSTALL_EXTRA_FLAGS="$EXTRA_FLAGS"
```
Comment (paulopontesm) [inline .circleci/config.yml]: shouldn't this eval be inside the makefile for the EXTRA_FLAGS?

I wont remember to do eval everywhere

**Comment by DReigada** [comment]
I might have a solution for this that does the eval inside the Makefile. I'm testing it

**Comment by DReigada** [comment]
Superseded by: #335

**Pull request closed without merging**